Cargo.toml

```diff
@@ -114,6 +113,23 @@ rstack-self = { version = "0.3.0", optional = true }
 [target.'cfg(not(target_env = "msvc"))'.dependencies]
 tikv-jemallocator = "0.5"
 
+[workspace.dependencies]
+futures = "0.3.30"
+futures-util = "0.3.30"
+parking_lot = { version = "0.12.1", features = ["deadlock_detection"] }
+prost = "0.11.9"
+prost-types = "0.11.9"
+schemars = { version = "0.8.16", features = ["uuid1", "preserve_order", "chrono", "url"] }
+serde = { version = "~1.0", features = ["derive"] }
+serde_cbor = { version = "0.11.2" }
+serde_json = "~1.0"
+tokio = { version = "~1.35", features = ["full"] }
+tokio-util = "0.7"
+tonic = { version = "0.9.2", features = ["gzip", "tls"] }
+tonic-reflection = "0.9.2"
+tracing = { version = "0.1", features = ["async-await"] }
+uuid = { version = "1.7", features = ["v4", "serde"] }
+
```

Comment:
The important bit is here. We define all these dependencies here only once now.

Reply:
I had no idea this was possible... 🤯